kafka_topic_enforcer/src/test/java/com/tesla/data/topic/enforcer/TopicEnforcerTest.java

```diff
@@ -48,16 +48,18 @@ public void testPartitionIncrease() {
         new ConfiguredTopic("a", 1, (short) 3, Collections.emptyMap()),
         // topic 'b' wants to increase partition count to 3
         new ConfiguredTopic("b", 3, (short) 1, Collections.emptyMap()),
-        new ConfiguredTopic("c", 1, (short) 1, Collections.emptyMap()));
+        new ConfiguredTopic("c", 3, (short) 3, Collections.emptyMap()));
 
     Map<String, ConfiguredTopic> existing = new HashMap<String, ConfiguredTopic>() {
       {
         // cluster topic has less replicas than configured but same partitions, we do not care!
         put("a", new ConfiguredTopic("a", 1, (short) 1, Collections.emptyMap()));
         // cluster topic has less partitions than configured, we care!
         put("b", new ConfiguredTopic("b", 1, (short) 1, Collections.emptyMap()));
+        // cluster topic has less partitions than configured and less replicas, we care (about the partitions)!
+        put("c", new ConfiguredTopic("c", 1, (short) 1, Collections.emptyMap()));
         // cluster has an extra topic that is not configured to be present, we do not care!
-        put("d", new ConfiguredTopic("b", 1, (short) 1, Collections.emptyMap()));
```

Comment:
what is wrong with name `d`?

Reply:
good question, not sure how this change came about, will revert